app/services/workflow.rb

```diff
@@ -8,14 +8,12 @@ class Workflow
   # @param [String] priority ('default') determines the relative priority used for the workflow.
   #                                      Value may be 'low' or 'default'
   def self.create_unless_exists(druid, workflow_name, version: 1, priority: 'default')
-    return unless workflow_client.workflow(pid: druid, workflow_name:).empty?
+    return if (workflow_client = workflow(druid, workflow_name)).present?
```

Comment:
I think you need to call `.find` on workflow client to determine if the workflow exists.